Automattic/jetpack
# [Plugin] Backup: Update error screen and detect backup retry

Detect when a backup will retry before showing error and update content of error screen.



#### Changes proposed in this Pull Request:

* Detect whenever a backup failed but will retry to display a screen showing that a backup will be ready soon.
* Modify the content of the error page and add a link to the support contact form on it. 

#### Jetpack product discussion
1201228005200946-as-1201228017959576

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
1- Create a test site and apply the plugin code on this PR
2- Run an initial backup and force an `error-will-retry` error. 
This can be done by interrupting the connection while the initial backup is running, deactivating the plugin or turning off jurassic.tube will do the work.
3- Re-establish connection and confirm that the plugin shows the `Your first cloud backup will be ready soon` page, without showing any progress bar.
4- Force a second fail. 
5- Once it fails again, confirm that the error page is shown and that the link to the support contact form works. 

Error page preview:
![Screen Shot 2021-11-13 at 00 04 02](https://user-images.githubusercontent.com/16329583/141593493-67d7fdc3-b0d4-4b9e-a4f8-190ddc73bb03.png)
